students/BRACH_Jakub/lesson_01_basics/L01P01_Three_Numbers.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+number_of_factors = 3
+summ = 0
+
+for x in range(0, number_of_factors):
+   import math
```

Comment:
Imports should be on top of the file : "Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants." from http://legacy.python.org/dev/peps/pep-0008/#imports

Besides, this one is not used anywhere

Reply:
First of all big THANKS for your time and so detailed review and remarks.

Code has been running locally on windows python 2.6. There were no errors, snakify tests has not been applied however. I'll look at this.

Is it possible to get stickler-ci fail reason? I haven't worked with this tool.

Reply:
Regarding linter review: 
> żeby to działało, w roocie na branchu, z którego robicie PR musicie mieć pliczek *.stickler.yml*. Jeżeli tego pliku nie macie, automat nie może sprawdzić poprawności i wystawia rejecta.".

So need to sync your fork with upstream branch to get this working.

In this course, we encourage to use Python 3.6.

Snakify tests are a great way to make sure that algorithm part is working correctly. Another benefit is that they can be run online.